clusterloader2/api/types.go

```diff
@@ -142,15 +142,28 @@ type TuningSet struct {
 	GlobalQPSLoad *GlobalQPSLoad `json: globalQPSLoad`
 }
 
+// MeasurementInstanceConfig is a structure that contains the Instance for wrapper measurements along with optional params.
+type MeasurementInstanceConfig struct {
+	// Identifier is an interface{} which could be a string or map of {name: value} that differentiates measurement instances of the same method.
+	Identifier string `json: identifier`
+	// Params is a map of {name: value} pairs which will be passed to the measurement method - allowing for injection of arbitrary parameters to it.
```

Comment:
Please add a comment on how conflicts between this and measurement.Params are resolved (these are preferred).